.gitignore

```diff
@@ -62,3 +62,5 @@ test/selenium/screenshots/*
 
 # logging
 logs/*.log*
+openassessment/xblock/static/css/openassessment-ltr.css.map
```

Comment:
Hmm, I'd prefer these not to be added. Can you update your local Sass compiler not to create sourcemaps?